eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -236,15 +238,35 @@ jobs:
           artifactName: '$(coreClrProductArtifactName)'
           displayName: 'CoreCLR product download for Mono'
 
+
+    # Build target specific tests
+    - ${{ if ne(parameters.corefxTests, true) }}:
```

Comment:
Nit - you should be able to combine the two if statements using the and() function and remove one level of nesting ;-).